src/tests/htif-yield.lua

```diff
@@ -19,17 +18,13 @@ where options are:
   --uarch-ram-image=<filename>
```

Comment:
We can probably remove this option in this test since the ram image is now embedded, right?

Reply:
I'm not so sure. Maybe one wants to run this test with an external image file.